server/channels/api4/bot_test.go

```diff
@@ -183,16 +197,19 @@ func TestPatchBot(t *testing.T) {
 		})
 		require.NoError(t, err)
 		CheckCreatedStatus(t, resp)
-		defer th.App.PermanentDeleteBot(th.Context, createdBot.UserId)
+		defer func() {
+			appErr := th.App.PermanentDeleteBot(th.Context, createdBot.UserId)
+			assert.Nil(t, appErr)
+		}()
 
 		th.TestForSystemAdminAndLocal(t, func(t *testing.T, client *model.Client4) {
 			botPatch := &model.BotPatch{
 				Username:    sToP(GenerateTestUsername()),
 				DisplayName: sToP("an updated bot"),
 				Description: sToP("updated bot"),
 			}
-			patchedBot, patchResp, err2 := client.PatchBot(context.Background(), createdBot.UserId, botPatch)
-			require.NoError(t, err2)
+			patchedBot, patchResp, appErr := client.PatchBot(context.Background(), createdBot.UserId, botPatch)
+			require.NoError(t, appErr)
```

Comment:
The return value of `client.PatchBot` is not a `model.AppError`, hence `appErr` is not the right variable name